src/irmin-pack/inode_intf.ml

```diff
@@ -83,6 +83,16 @@ module type INTER = sig
 
     val integrity_check : t -> bool
     (** Checks the integrity of an inode. *)
+
+    val to_tree :
```

Comment:
Could you simplify the exposed type a little bit?

e.g. something like 
```ocaml
type tree :=  [`Tree of hash * tree option list | `Values of hash * (step * value) list ]

type to_tree : t -> tree
```

You could also simplify the documentation to just say that the function exposes the underlying (hashed) inode tree.